requirements/base.txt

```diff
@@ -26,3 +26,20 @@ xlrd==0.9.3
 
 # images / thumbnails
 sorl-thumbnail==12.3
+
+# CMS-related
+django-cms==3.1.3
+  djangocms-admin-style==0.2.2
+  django-classy-tags==0.6.2
+  django-treebeard==3.0
+  html5lib==0.999
+  django-sekizai==0.8.2
+cmsplugin-filer==0.10.2
+  easy-thumbnails==2.2
+  django-filer==0.9.12
+    django-mptt==0.6.1
+    django-polymorphic==0.7.2
+    Unidecode==0.4.18
+  django-appconf==1.0.1
+https://github.com/divio/djangocms-text-ckeditor/archive/master.zip?1445623824.9186497
```

Comment:
Not familiar with this syntax. Is this pinned to a commit or will it change as master changes?